bin/tact

```diff
@@ -165,9 +165,9 @@ meowModule.then(
         projectNames: cli.flags.projects ?? [],
         additionalCliOptions: { mode },
       })
-      .then((success) => {
+      .then((response) => {
```

Comment:
Two points here:
1. It would be nice to add a `--quiet` flag, which would be passed to `suppressLog: ...` argument of `run()`
2. If there's no `--quiet` flag, it would be nice to peek into `response.error` and display them onto console before we do `process.exit(...)`

Other than that, splendid work!

Reply:
@novusnota can you please help with this flag since you already have experience with meow? just push your commit to this PR

Reply:
Sure!

Reply:
Hmm, seems like adding `response.error` doubles the error output:

```ts
// bin/tact
// ...
main
  .run({
    fileName: cli.input.at(0),
    configPath: cli.flags.config,
    projectNames: cli.flags.projects ?? [],
    additionalCliOptions: { mode },
    suppressLog: cli.flags.quiet
  })
  .then((response) => {
    // Log errors if --quiet is not set
    if (!cli.flags.quiet) {
      response.error.map(err => console.log(err)); // ← this logs errors AFTER the main log output with them showed already, which isn't nice
    }
    // https://nodejs.org/docs/v20.12.1/api/process.html#exit-codes
    process.exit(response.ok ? 0 : 30);
  });
```

Not a big deal though, can omit the `reponse.error.map...`